### PR TITLE
Re-export Logging.jl

### LINKED ISSUE #43: [Request] Reexport Logging?
There's never a time where I do `using LoggingExtras` that I don't also have to do `using Logging`. Would there be any objection to using Reexport (Or, since it's a Stdlib and there [aren't all that many exports](https://github.com/JuliaLang/julia/blob/master/stdlib/Logging/src/Logging.jl#L32-L46), maybe just do it mannually)? I'd be happy to make a PR, but didn't want to be presumptuous.

### PR DESCRIPTION
**Pull request by kescobo**
Closes #43 

I think I did this right. Probably could use some documentation, but wasn't totally sure where.

### CODE REVIEW

File: src/LoggingExtras.jl [hunk 1]
```diff
@@ -12,6 +12,17 @@ export TeeLogger, TransformerLogger, FileLogger,
     ActiveFilteredLogger, EarlyFilteredLogger, MinLevelLogger,
     DatetimeRotatingFileLogger, FormatLogger
 
+######
+# Re export Logging.jl from stdlib 
+# list is stable between julia 1.0 and 1.6
+# https://github.com/JuliaLang/julia/blob/release-1.6/stdlib/Logging/src/Logging.jl#L32-L46
+import Logging: @debug, @info, @warn, @error, @logmsg,
+    with_logger, current_logger, disable_logging, ConsoleLogger
+
```
Comment (oxinabox) [inline src/LoggingExtras.jl:21]: Should bne enough just to bring them in scope via `using`
```suggestion
using Logging
```

**Comment by oxinabox** [comment]
>  Probably could use some documentation, but wasn't totally sure where.

Just under [usage](https://github.com/oxinabox/LoggingExtras.jl#usage).

> Load the package with using LoggingExtras. You likely also want to load the Logging standard lib. Loggers can be constructed and used like normal.

should be changes to say _for convienience Logging Extras re-exports the Logging standard library"_ or something like that

File: src/LoggingExtras.jl (hunk 1 above)
Comment (kescobo) [reply src/LoggingExtras.jl:21]: Huh, OK. This does work - is the `import` only needed if you're going to add methods?

**Comment by kescobo** [comment]
OK, made those changes, and also removed all of the `using Logging` from examples. Turns out that I also needed to add `export Logging` in order to use things like `Logging.Debug`.

**Comment by kescobo** [comment]
@oxinabox bump :-)

File: src/LoggingExtras.jl (hunk 1 above)
Comment (oxinabox) [reply src/LoggingExtras.jl]: correct

**Comment by oxinabox** [comment]
Thanks.
Sorry for the delay, i have been super busy

> OK, made those changes, and also removed all of the using Logging from examples. Turns out that I also needed to add export Logging in order to use things like Logging.Debug.

I am not sure i love this. Feels odd to have a module itself reexported.
But we can always remove it later it it somehow causes problems.
(seems very unlikely too)

**Pull request merged**